src/sagemaker/jumpstart/types.py

```diff
@@ -1236,8 +1106,11 @@ class DescribeHubResponse(JumpStartDataHolderType):
         "hub_status",
         "last_modified_time",
         "s3_storage_config",
+        "_region",
```

Comment:
can you create am abstract class for api input/output response payload, and make `region` a field of that?